.travis.yml

```diff
@@ -5,3 +5,4 @@ node_js:
 script:
   - npm run lint
   - npm run dist
```

Comment:
Also remove this script, cuz we don't have such rule in our package.json. Travis failed because of this.

Reply:
`npm run dist`

Reply:
Oh my bad!